docs/source/workshops/bioc-2015.rst

```diff
@@ -45,114 +32,87 @@ Install gcloud
 Configure gcloud compute
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. container:: toggle
-
-    .. container:: header
-
-        There are a few steps to do this.  Click here to **Show/Hide Instructions**
-
-    .. container:: content
-
-      .. include:: /includes/gcloud_compute_configure.rst
+.. include:: /includes/gcloud_compute_configure.rst
 
 Spin up a Google Compute Engine Instance running Bioconductor
 -------------------------------------------------------------
 
-This codelab can be run from anywhere.  All the heavy lifting is happening in the cloud regardless of where R is running.  But if you like, you can run the whole darn thing in the cloud.
+This codelab `can be run from anywhere <https://github.com/googlegenomics/bioconductor-workshop-r>`_.  All the heavy lifting is happening in the cloud regardless of where R is running.  But if you like, you can run the whole darn thing in the cloud.
```

Comment:
Personal preference to be less colloquial. Instead of "you can run the whole darn thing in the cloud", is it worth mentioning something about GCE providing a consistent and tested environment?